scripts/mkdist.js

```diff
@@ -14,43 +14,40 @@ function mkdist (opts, cb) {
 
   var dist = opts.dist
   var ver = dist.version
-  if (ver.match(/^v/)) ver = ver.substring(1)
```

Comment:
This has to be a concrete version. It doesnt really matter if you "dont have a release process". Just pick a version and tag it.

The point here being that anybody should be able to pull down and build dists and end up with the same resultant binaries (ideally, though in reality builds arent always deterministic)